build.gradle.kts

```diff
@@ -23,6 +24,15 @@ plugins {
 	id("com.github.johnrengelman.shadow") version "5.0.0"
 	id("com.github.breadmoirai.github-release") version "2.2.9"
 	id("com.github.ben-manes.versions") version "0.21.0"
+	id("com.install4j.gradle") version "8.0"
+}
+
+install4j {
+	var install4jDir : String? = null
```

Comment:
I think these 5 lines can be consolidated into less, this looks a bit bloated.

Reply:
29118a42f0055903ea14d19235d7d3ffa8519465